packages/react-workers/src/workers/methods.js

```diff
@@ -236,11 +239,26 @@ export function getRawFeatures({
   let hasData = false;
 
   if (currentFeatures) {
-    rows = applySorting(getFilteredFeatures(filters, filtersLogicalOperator), {
-      sortBy,
-      sortByDirection,
-      sortByColumnType
-    });
+    const filteredFeatures = applySorting(
+      getFilteredFeatures(filters, filtersLogicalOperator),
+      {
+        sortBy,
+        sortByDirection,
+        sortByColumnType
+      }
+    );
+
+    if (searchFilterColumn && searchFilterText) {
```

Comment:
Minor. I would apply sorting after searching.
Searching is kind-a filtering. Is it?